src/main/java/github/scarsz/discordsrv/objects/threads/ServerWatchdog.java

```diff
@@ -66,7 +68,10 @@ public void run() {
                         continue;
                     }
 
-                    String channelName = DiscordSRV.getPlugin().getMainTextChannel().getName();
+                    @SuppressWarnings("ConstantConditions" /* getDestinationTextChannelForGameChannelName may return null */)
```

Comment:
Since there is only one suppression it'd look neater to have it like this (also matches existing code style better):
```java
@SuppressWarnings("ConstantConditions") // comment
```

Reply:
I will make this change to comply with your style guidelines. The reason I prefer the comment inside of the annotation, is that my preferred IDE (IntelliJ IDEA), will collapse the comment as pictured in the attachment, resulting in a neater view. Clicking on the collapsed annotation expands it and shows the comment when necessary.

Comment inside:
![image](https://user-images.githubusercontent.com/20528490/88432264-daba8980-cdc9-11ea-80c3-d5418ce8ba68.png)

Comment outside:
![image](https://user-images.githubusercontent.com/20528490/88432273-dee6a700-cdc9-11ea-8e2e-ca2def8cbffa.png)